### ISSUE TITLE
Unable to resize Toolbar.

### ISSUE THREAD

**Issue by GJNelson7030** [linuxgurugamer/ksp_toolbar#23]
I've got ToolBar v1.8.0.5 loaded, using KSP v1.8.1, and ClickThrough Blocker v0.1.9.3 loaded. When I choose the Unlock Position and Size" option, I can not resize the ToolBar, only change it's position. If I'm doing something, I don't know what it is. I've used this mod in the past and not had problems. I recently started playing KSP again.

**Comment by GJNelson7030** [comment]
Ok, now I discovered if the ToolBar is empty I can resize it, then "Lock" it again and add buttons that keep adding them in one long row in the ToolBar. I can only resize an empty ToolBar.

**Comment by GJNelson7030** [comment]
Another update. When the toolbar is empty, I can resize it to any shape, lock it again, then when I add buttons to be visible, they are added to the shape restrictions from when I resized the toolbar when it was empty. How could I have screwed this up so bad? LoL!!! Any help would be appreciated.

**Comment by GJNelson7030** [comment]
Ok, I'm not sure which, but I've found it was not ToolBar that caused my problems. I removed the KSP v1.8.1 outdated mods Kerbal Wind Tunnel and RCS Build Aid Continued, uninstalled and reinstalled ToolBar, and rechecked how ToolBar was operating. Seems to be normal now.

It might have been Kerbal Wind Tunnel, since at one point it was showing the wrong icon for it on the ToolBar. It was labelled as ReCoupler and colored a weird purple color. Clicking on it brought up Kerbal Wind Tunnel though.

Also, while investigating my problems, the Stockalike Station Parts Expansion Redux (SSPX Redux) got an update. It was throwing errors in the KSP.Log, but seemed to be working correct. It's change log entry said something about correcting a missing [ (square bracket) symbol that was missing in one of it's files. Here is the entry from it's change log...

"Fixed a missing bracket."

Loading KSP, it's no longer spitting out the errors (there were 20 of them) that it was previously.

So, I think something corrected the problem and I'll keep you updated if it returns. Thanks for reading this!

**Comment by linuxgurugamer** [comment]
Purple means missing texture.
Glad you figured it out

**Comment by GJNelson7030** [comment]
I goofed in stating the mods I removed. I removed Kerbal Wind Tunnel and SmartStage, not RCS Build Aid Continued. Plus I found out the real culprit.

Of all things it was Kerbal Construction Time (KCT) that was causing the problems. When I use Toolbar Controller to put KCT on Blizzy's Toolbar, I can not resize the Toolbar correctly. When I use Toolbar Controller to put KCT back on the Stock Toolbar, the problems go away.

I'm going to forward this info to the caretaker for KCT, but wanted to let you know more info on what I found out, in case someone else came to you with this problem.
